### ISSUE TITLE
Redundant labeling of the menu button

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria-practices#1797]
[Navigation Menu Button Example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/menu-button/menu-button-links.html): The menu refers to the button label via aria-labelledby. This leads to the fact that after activating the button, you hear its label with the screen reader again (as the menu label). I think this is redundant - and a menu label should be avoided

**Comment by mcking65** [comment]
@JAWS-test 

This is intentional.

It is important that menus have labels and be perceivable as menus, especially as we strive to improve experiences for touch and reading cursors.

**Comment by mcking65** [comment]
Closing. Feel free to re-open if you disagree. However, if you don't believe labeling of menus should be supported, that would be a spec issue.

**Comment by JAWS-test** [comment]
I find it useful that menus can be labeled. I just don't think it makes sense in this specific case, because the menu label is derived indirectly from the button label, which is output by the screen reader immediately before
